core/arch/arm/dts/stm32mp135f-dk.dts

```diff
@@ -87,6 +298,7 @@
 		CLK_STGEN_HSE
 		CLK_USBPHY_HSE
 		CLK_I2C4_HSI
+		CLK_I2C5_HSI
```

Comment:
Won't this remain unused?

Reply:
I meant we do not have a use case but I guess it's completely fine for synchro

Reply:
st,stm32mp13-rcc-mco => not domcuement in yaml file, no reason to change compatible 

I prefer a other solution, can you drop the MCO part, I will propose something soon

Patrick

Reply:
@patrickdelaunay, I think your comment is misplaced and adding `CLK_I2C5_HSI` source clock is not related to MCO. Please tell me if i'm wrong.